src/htrun/README.md

```diff
@@ -261,9 +261,9 @@ Key-value protocol was developed and is used to provide communication layer betw
 
 # Greentea client API
 
-DUT test API was first introduced in ```mbedmicro/mbed``` project [here](https://github.com/mbedmicro/mbed/tree/master/libraries/tests/mbed/env). After refactoring this functionality was copied and improved in [greentea-client](https://github.com/ARMmbed/mbed-os/tree/master/features/frameworks/greentea-client) module.
+[greentea-client](https://github.com/ARMmbed/greentea-client) is a C++ client library and provides APIs which can be used to communicate with the Greentea host test tool.
 
-* Slave side key-value protocol API, see [here](https://github.com/ARMmbed/mbed-os/blob/master/features/frameworks/greentea-client/greentea-client/test_env.h) for details.
+* Slave side key-value protocol API, see [here](https://github.com/ARMmbed/greentea-client/blob/master/include/greentea-client/test_env.h) for details.
 
 ```c++
```

Comment:
I think the two code blocks below can be removed, and the paragraphs can be reworded saying `greentea_send_kv()` sends key-value pairs, `greentea_parse_kv` ... As we already have a link to `test_env.h`, there's no need to have an inline copy of the the API in my opinion, as we would need to keep them updated if anything changes in the future.